packages/hoppscotch-app/components/app/PaneLayout.vue

```diff
@@ -56,5 +61,73 @@ const SIDEBAR = useSetting("SIDEBAR")
 
 const slots = useSlots()
 
+let panelVerticalInfo: any = [
+  {
+    max: 100,
+    min: 0,
+    size: 75,
+  },
+  {
+    max: 100,
+    min: 0,
+    size: 65,
+  },
+]
+
+let panelHorizontalInfo: any = [
+  {
+    max: 100,
+    min: 0,
+    size: 75,
+  },
+  {
+    max: 100,
+    min: 0,
+    size: 65,
+  },
+]
+
+function setVerticalPanelInfo(event: any) {
```

Comment:
There should be an explicit type for this event. If there's no explicit type then we should put `unknown` as type.